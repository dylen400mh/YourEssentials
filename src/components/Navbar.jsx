import { Link } from "react-router-dom";

function Navbar() {
  return (
    <nav data-testid="Navbar">
      <Link to="/">Home</Link>
      <Link to="shop">Shop</Link>
    </nav>
  );
}

export default Navbar;
