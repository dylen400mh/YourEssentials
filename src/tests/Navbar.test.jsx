/* eslint-disable no-undef */
import { render } from "@testing-library/react";
import Navbar from "../components/Navbar";
import { MemoryRouter } from "react-router-dom";

describe("Navbar component", () => {
  it("Renders a navbar with home and shop links", () => {
    const { container } = render(
      <MemoryRouter>
        <Navbar />
      </MemoryRouter>
    );

    expect(container).toMatchSnapshot();
  });
});
