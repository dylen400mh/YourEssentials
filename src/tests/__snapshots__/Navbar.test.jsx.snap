// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Navbar component > Renders a navbar with home and shop links 1`] = `
<div>
  <nav
    data-testid="Navbar"
  >
    <a
      href="/"
    >
      Home
    </a>
    <a
      href="/shop"
    >
      Shop
    </a>
  </nav>
</div>
`;
